tests/unit/amo/components/TestIconPromotedBadge.js

```diff
@@ -67,30 +59,33 @@ describe(__filename, () => {
   ])(
     'adds an alt property for category="%s" when showAlt is true',
     (category, alt) => {
-      const root = render({ category, showAlt: true });
+      render({ category, showAlt: true });
 
-      expect(root.find(Icon)).toHaveProp('alt', alt);
+      expect(screen.getByText(alt)).toBeInTheDocument();
```

Comment:
another `screen.getByAltText`?

Reply:
As above, `getByAltText` doesn't find anything because the text isn't being rendered in an actual `alt` property, it's just in a `span`.